01_run_pancancer_classification.py

```diff
@@ -0,0 +1,49 @@
+"""
+Script to run pancancer classification experiments, by calling classify_cancer_type.py
+with the corresponding arguments
+"""
+import os
+import sys
+import subprocess
+
+import pandas as pd
+from tqdm import tqdm
+
+import pancancer_utilities.config as cfg
+
+EXP_SCRIPT = os.path.join(cfg.repo_root,
+                          'pancancer_utilities',
+                          'scripts',
+                          'classify_cancer_type.py')
+
+# genes and cancer types to run experiments for
+# just hardcoding these for now, might choose them systematically later
+genes = ['TP53', 'PTEN', 'KRAS', 'BRAF', 'TTN']
+cancer_types = ['BRCA', 'THCA', 'SKCM', 'GBM', 'SARC']
+
+def run_single_experiment(gene, cancer_type, use_pancancer,
+                          shuffle_labels, verbose=False):
+    args = [
+        'python',
+        EXP_SCRIPT,
+        '--gene', gene,
+        '--holdout_cancer_type', cancer_type
+    ]
+    if use_pancancer:
+        args.append('--use_pancancer')
+    if shuffle_labels:
+        args.append('--shuffle_labels')
+
+    if verbose:
+        print('Running: {}'.format(' '.join(args)))
+    subprocess.call(args)
+
+if __name__ == '__main__':
```

Comment:
Still new to pipeline building. This seems interesting to me, I wonder if you could walk me through the logic here. So you have this main function that calls this `run_single_experiment`, which generates a python terminal call...is there a reason to have it as a terminal call?

Reply:
Not a good reason, other than that a lot of this code existed as a script before so it was easiest. I think longer-term it would be best to refactor this file to be a series of functions, which I could then just import into a script like this (rather than having to use `subprocess`).

This would also have the advantage that I'd only have to load all of the data once, as opposed to loading it each time I call the script. I have #7 to remind me to make this change in the future.